docs/using-the-jdbc-driver/UsingTheJdbcDriver.md

```diff
@@ -100,6 +100,25 @@ DriverConfigurationProfiles.addOrReplaceProfile(
         CustomConnectionPluginFactory.class));
 ```
 
+### Connection Initialization
+In some cases it's necessary to configure a connection before a user application can use it. Some target drivers provides such functionality and allow to specify a configuration parameter with SQL statements that are executed when connection is established. However, not all drivers supports such functionality. Also, in some cases, additional conditions should be checked in order to identify what initialization is required for a particular connection.
```

Comment:
In some particular cases, users may need to define a specific configuration for a connection that has just been opened by the driver before an application can use it.

Some...